simulation/traffic_simulator/include/traffic_simulator/entity/entity_manager.hpp

```diff
@@ -290,6 +291,8 @@ class EntityManager
 
   bool isEgo(const std::string & name) const;
 
+  const std::string getEgoName() const;
+
```

Comment:
This file is intentionally formatted so that the member function names are aligned.
Rewrite as follows according to the format of other member functions.

```
auto getEgoName() const -> const std::string;
```